Geometry/HGCalCommonData/src/HGCalWaferMask.cc

```diff
@@ -1278,22 +1278,107 @@ std::vector<std::pair<double, double> > HGCalWaferMask::waferXY(
 	Depending on the wafer type and placement index, the corners
 	are chosen in the variable *np*
   */
-  double dx[24] = {HGCalTypes::c00 * delX,  HGCalTypes::c10 * delX,  HGCalTypes::c10 * delX,  HGCalTypes::c00 * delX,
+  double Sin_60 = std::sqrt(3.0) / 2.0;
+  double Cos_60 = 0.5;
+  double Tan_60 = Sin_60 / Cos_60;
+  double delX = 0.5 * waferSize;
```

Comment:
If wafersize can be passed as a `const`, also all these values and vectors could be made const